requirements/runtime.txt

```diff
@@ -1,5 +1,6 @@
 matplotlib
 numpy
+panopticapi @ git+https://github.com/cocodataset/panopticapi.git
```

Comment:
Need to consider whether it must be installed, especially for window users.

Reply:
Checked.
It can be installed successfully on both Ubuntu and Windows.